.github/workflows/kv-test-failure-stale.yml

```diff
@@ -0,0 +1,34 @@
+name: Mark stale test failure issues for KV
+
+on:
+  schedule:
+  - cron: "0 10 * * 1-4"
+  workflow_dispatch:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+    - uses: actions/stale@v3
+      with:
+        operations-per-run: 1000
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: |
+           We have marked this test failure issue as stale because it has been 
+           inactive for 1 month. If this failure is still relevant, removing the 
+           stale label or adding a comment will keep it active. Otherwise, 
+           we'll close it in 5 days to keep the terst failure queue tidy. 
+        stale-pr-message: 'Stale pull request message'
+        stale-issue-label: 'no-test-failure-activity'
+        stale-pr-label: 'no-pr-activity'
+        close-issue-label: 'X-stale'
+        close-pr-label: 'X-stale'
+        # Disable this for PR's, by setting a very high bar
+        days-before-pr-stale: 99999
+        days-before-issue-stale: 30
+        days-before-close: 5
+        only-labels: 'T-KV,C-test-failure'
```

Comment:
as mentioned, this results in excluding all issues, since T-kv is somehow not set on test failures when the project is assigned.

Reply:
hmm, but then we would need to enable this for all test failures, which would need more communication. The bot does not understand github projects, so it's all label based.

Reply:
Can we find out why T-KV is not set for the test failures? Our friendly local exalate experts would know (not exactly sure who that is, James from Dev-Inf?). Maybe it's as simple as tweaking the exalate config.

Reply:
It looks like Oliver made a change in blathers to fix this